fdb-extensions/src/main/java/com/apple/foundationdb/clientlog/DatabaseClientLogEvents.java

```diff
@@ -62,10 +59,10 @@
      */
     @FunctionalInterface
     public interface RecordEventConsumer {
```

Comment:
I'm not sure `RecordEventConsumer` (instead of just `EventConsumer`) makes sense now that it's in `fdb-extensions`.

Reply:
Changed.